### PR TITLE
Add moira 4.0.0.3.

### PR DESCRIPTION
**Pull request by leee**
_If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so._

After making all changes to the cask:
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:
- [x] Named the cask according to the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked that the cask was not already refused in [closed issues](https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed).

Signed-off-by: Eugene Lee leee@mit.edu

### CODE REVIEW

File: Casks/moira.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
+cask 'moira' do
+  version '4.0.0.3'
+  sha256 'e629324916533e9651d6cc0c9318361fbaaa06bb75fb722146ca51cd62251ff1'
+
+  url "https://github.com/macathena/moira/archive/#{version}.zip"
```
Comment (reitermarkus) [inline Casks/moira.rb]: This needs an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

File: Casks/moira.rb [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  name 'MIT Project Athena (Macathena) - Moira for Apple macOS/OS X'
+  homepage 'https://github.com/macathena'
+  license :unknown
+
+  depends_on formula: ['homebrew/dupes/krb5']
```
Comment (reitermarkus) [inline Casks/moira.rb]: You don't need an array here.

File: Casks/moira.rb [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  artifact "moira-#{version}/man/man8/moirad.8", target: '/usr/local/share/man/man8/moirad.8'
+  artifact "moira-#{version}/man/man8/mrtest.8", target: '/usr/local/share/man/man8/mrtest.8'
+  artifact "moira-#{version}/man/man8/reg_svr.8", target: '/usr/local/share/man/man8/reg_svr.8'
+  artifact "moira-#{version}/man/man8/setquota.8", target: '/usr/local/share/man/man8/setquota.8'
+  artifact "moira-#{version}/man/man8/startdcm.8", target: '/usr/local/share/man/man8/startdcm.8'
+  artifact "moira-#{version}/man/man8/startmoira.8", target: '/usr/local/share/man/man8/startmoira.8'
+  artifact "moira-#{version}/man/man8/startreg.8", target: '/usr/local/share/man/man8/startreg.8'
+  artifact "moira-#{version}/man/man8/update_server.8", target: '/usr/local/share/man/man8/update_server.8'
+
+  caveats 'You MUST use MIT Kerberos as opposed to Apple-provided Heimdal Kerberos.'
```
Comment (reitermarkus) [inline Casks/moira.rb]: Use only one `caveats` stanza with `<<-EOS.undent`.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

**Comment by fanquake** [comment]
@leee Can you make the requested changes?

File: Casks/moira.rb (hunk 3 above)
Comment (leee) [inline Casks/moira.rb]: Done.

File: Casks/moira.rb (hunk 2 above)
Comment (leee) [inline Casks/moira.rb]: Modified - this was blocked until `hesiod` was added and updated in `homebrew-core`.

File: Casks/moira.rb (hunk 1 above)
Comment (leee) [inline Casks/moira.rb]: Done - didn't know about appcast - thank you!

**Review by leee — COMMENTED** [review]
@fanquake - apologies for the delay. There were updates that had to be made upstream as well as a package provided for in `homebrew-core`.

**Comment by leee** [comment]
The [macathena](https://github.com/macathena) team has been provided a public source from upstream and therefore find this PR moot/no longer necessary, and intend on writing a Formula for use in [homebrew-core](https://github.com/homebrew/homebrew-core).

Thank you for the initial code review and consideration for inclusion!

**Pull request closed without merging**